docs/user-guide.md

```diff
@@ -18,9 +20,11 @@ The requirements to run this user guide are the following:
 
 The architecture from Asperathos is depicted in the following figure. The platform follows the REST architecture for submissions and is composed by three main modules and one optional module: **(i)** the Manager is the entry point for the user and is responsible for receiving an application submission, triggering all other steps; **(ii)** the Monitor gather, transform and publish metrics collected from applications (e.g., the application progress) or environment resources (e.g., CPU usage); **(iii)** the Controller is the component that adjusts the amount of allocated resources dedicated to an application; **(iv)** the Visualizer, the unique optional component of the system, leverages existing tools to provide rich visualizations of the applications being run by Asperathos. 
 
-![Asperathos Architecture](imgs/asperathos-archtecture.png)
+![Asperathos Architecture](../imgs/asperathos-archtecture.png)
+
+Each module above can be customized by plugins. For example, a plugin to deploy the application can be specified in the Manager, another to define the application specific metric can be specified in the Monitor, another to control the actuation logic could be configured in the Controller, another to launch a specific visualization platform and so on.
 
-Each module above can be customized by plugins. For example, a plugin to deploy the application can be specified in the Manager, another to define the application specific metric can be specified in the Monitor, another to control the actuation logic could be configured in the Controller, another to launch a specific visualization platform and so on. 
+![Asperathos Containers](../imgs/containers-asperathos.png)
```

Comment:
In the image, correct label: "Controls scailing" -> "Controls scaling"

Reply:
Labels fixed. Image changed in commit 59cdc4d104b8a0e7ba3f89aab68eac8ee3d714b7